### ISSUE TITLE
Possibility to change Wind direction from arrows to text

### ISSUE THREAD

**Issue by Lamandus** [CleverRaven/Cataclysm-DDA#35819]
**Is your feature request related to a problem? Please describe.**  
I like seeing the "Wind: light breeze ⇗" in my UI. The Problem is, only Unifont is actually supporting it right now. Even Terminus is lacking those arrows,

**Describe the solution you'd like**  
"Wind: light breeze NE" Instead of using arrows, the compassorientation will be written. 

**Describe alternatives you've considered**  
Maybe using ← arrows instead, since these are more common.

**Additional context**  
I mean, I can just switch off "wind" from the UI and be done with it. But I rather would like using it still and seeing where the Wind is flowing.

**Comment by Qrox** [comment]
You can now change `config/fonts.json` to specify fallback fonts to display these symbols. The format is `"typeface": [ "main font", "fallback font (such as unifont)" ]`.

**Comment by Lamandus** [comment]
> 
> 
> You can now change `config/fonts.json` to specify fallback fonts to display these symbols. The format is `"typeface": [ "main font", "fallback font (such as unifont)" ]`.

wow! Superb! Thx!
